core/server/models/post.js

```diff
@@ -530,14 +530,21 @@ Post = ghostBookshelf.Model.extend({
         };
     },
 
-    orderDefaultRaw: function () {
-        return '' +
+    orderDefaultRaw: function (options) {
+        let order = '' +
             'CASE WHEN posts.status = \'scheduled\' THEN 1 ' +
             'WHEN posts.status = \'draft\' THEN 2 ' +
             'ELSE 3 END ASC,' +
             'CASE WHEN posts.status != \'draft\' THEN posts.published_at END DESC,' +
             'posts.updated_at DESC,' +
             'posts.id DESC';
+
+        // CASE: if the filter contains an `IN` operator, we should return the posts first, which match both tags
+        if (options.filter && options.filter.match(/(tags|tag):\s?\[.*\]/)) {
```

Comment:
`\s` -> TIL we support white space in filters :sweat_smile:

Reply:
i saw your test case here: https://github.com/TryGhost/Ghost/pull/10166/files#diff-5a0b10849d1a90717049d92c728caaa5R49

maybe we should remove this whitespace for now 🤔 
i am not 100% sure if it works, have not tested it on a theme

Reply:
nql-lang picks it up, but testing in real-world scenario would def make things clear

Reply:
i think for now: doesn't matter much because its optional anyway in the regex?

Reply:
agreed. let's merge this, looks good